<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialSettingWidget</class>
 <widget class="QWidget" name="SerialSettingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>161</width>
    <height>50</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel
{
	font: 12px &quot;Helvetica&quot;;
	color: rgb(255, 255, 255);
}</string>
  </property>
  <widget class="QLabel" name="labelBoard">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>10</y>
     <width>113</width>
     <height>13</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>CoolMakers Arduino Starter</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSerial">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>30</y>
     <width>113</width>
     <height>13</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelImage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>12</y>
     <width>26</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
